assets/js/lib/key_buffer.js

```diff
@@ -31,11 +31,8 @@ class KeyBuffer {
    * Immediately clears the buffer.
    */
   reset() {
-    if (this.resetTimeout) {
-      clearTimeout(this.resetTimeout);
-    }
-
-    this.clearTimeout = null;
+    clearTimeout(this.resetTimeoutId);
```

Comment:
`null` is silently ignored, but I think conditional is better for clarity :)

```suggestion
    if (this.resetTimeoutId) {
      clearTimeout(this.resetTimeoutId);
    }

```